src/utils/sign-images-in-ans-object/index.js

```diff
@@ -0,0 +1,41 @@
+const signImagesInANSObject =
+	(cachedCall, fetcher, resizerAppVersion, cacheKey = "image-token") =>
+	({ data, ...rest }) => {
+		const replacements = new Set();
+
+		const stringData = JSON.stringify(data, (key, value) => {
```

Comment:
I'm very impressed w/ using the `replacer` parameter in `JSON.stringify`. I was wondering if it would account for nested data, but it checked out for me.

Reply:
The replacer function is natively recursive and it starts at the root node so even if you pass in an ANS Image object, inspecting the value object still captures the need here.